packages/synapse-react-client/src/lib/style/components/_governance-dashboard.scss

```diff
@@ -79,16 +79,17 @@
       margin-bottom: 0;
     }
 
-    .ProjectIdInputGroup {
-      display: flex;
-      gap: 5px;
+    label {
+      margin-top: 0;
+      margin-bottom: 5px;
     }
 
-    .styled-svg-wrapper {
-      position: absolute;
-      right: 5px;
-      z-index: 5;
-      margin-top: -2px;
+    .UserSearchBoxV2,
+    .AsyncSelect {
+      .form-control {
+        background-color: map.get(SRC.$colors, 'gray-200');
+        border-radius: 3px;
```

Comment:
Add styling so that UserSearchBoxV2 and AsyncSelect styling is passably similar next to MUI TextField in the ReviewerDashboard. In the future, update those components to have styling in line with the MUI theme

Reply:
True! I've generalized the styling in the next commit